File: Command.java
import java.lang.*;

class Command
{
 public static  void main(String Arg[])throws Exception

{
 System.out.println("First number is"+Arg[0]);
 System.out.println("First number is"+Arg[1]);

 int ans = Integer.parseInt(Arg[0]) + Integer.parseInt(Arg[1]);
 System.out.println("Addition is :" +ans);

}

}

//javac command.java
//java Command 10 11

File: Command.c
#include<stdio.h>
#include<stdlib.h>

int main(int argc, char*argv[])//secont arg is array of char pointer
 {
  int ans = 0;
  printf("name of executable is : %s\n",argv[0]);
  printf("number of command line arguments are: %d\n",argc);
  
 
  ans = atoi(argv[1]) + atoi(argv[2]);  //asci to integer
   printf("Addition is : %d\n",ans);


  return 0;

 }

//gcc Command./c -o Myexe
//./Myexe 11 21